src/ngraph_mark_for_clustering.cc

```diff
@@ -706,6 +471,25 @@ bool NodeIsMarkedForClustering(const Node* node) {
           is_marked);
 }
 
+void GetStaticInputs(const Node* node, std::vector<int32>* inputs) {
+  if (GetNodeAttr(node->attrs(), "_ngraph_static_inputs", inputs) != Status::OK()) {
+    *inputs = std::vector<int32>{};
+  }
+}
+
+bool InputIsStatic(const Node* node, int index) {
+  std::vector<int32> inputs;
+  GetStaticInputs(node,&inputs);
+
+  for (auto i : inputs) {
```

Comment:
The for loop can be replaced by a 'find' line"
return std::find(inputs.begin(), inputs.end(), index)!=inputs.end();